docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md

```diff
@@ -282,15 +280,15 @@ The remainder of the HelloProjectOData.js file includes two functions: the `retr
 1. Delete all code in the default HelloProjectOData.js file, and then add the global variables and `Office.initialize` function. Variable names that are all capitals imply that they are constants; they are later used with the `_pwa` variable to create the REST query in this example.
 
     ```js
-    var PROJDATA = "/_api/ProjectData";
-    var PROJQUERY = "/Projects?";
-    var QUERY_FILTER = "$filter=ProjectName ne 'Timesheet Administrative Work Items'";
-    var QUERY_SELECT1 = "&amp;$select=ProjectId, ProjectName";
-    var QUERY_SELECT2 = ", ProjectCost, ProjectWork, ProjectPercentCompleted, ProjectActualCost";
-    var _pwa;           // URL of Project Web App.
-    var _projectUid;    // GUID of the active project.
-    var _docUrl;        // Path of the project document.
-    var _odataUrl = ""; // URL of the OData service: http[s]://ServerName /ProjectServerName /_api/ProjectData
+    let PROJDATA = "/_api/ProjectData";
+    let PROJQUERY = "/Projects?";
+    let QUERY_FILTER = "$filter=ProjectName ne 'Timesheet Administrative Work Items'";
+    let QUERY_SELECT1 = "&amp;$select=ProjectId, ProjectName";
+    let QUERY_SELECT2 = ", ProjectCost, ProjectWork, ProjectPercentCompleted, ProjectActualCost";
+    let _pwa;           // URL of Project Web App.
+    let _projectUid;    // GUID of the active project.
+    let _docUrl;        // Path of the project document.
+    let _odataUrl = ""; // URL of the OData service: http[s]://ServerName /ProjectServerName /_api/ProjectData
 
     // The initialize function is required for all add-ins.
     Office.initialize = function (reason) {
```

Comment:
Should we / Can we convert this to Office.ready?

Reply:
Good call. Updated!